crates/aptos-rosetta/src/types/objects.rs

```diff
@@ -427,10 +427,12 @@ pub struct Transaction {
     pub metadata: TransactionMetadata,
 }
 
-#[derive(Clone, Copy, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct TransactionMetadata {
     pub transaction_type: TransactionType,
     pub version: U64,
+    pub failed: bool,
```

Comment:
Why not success: bool?

Reply:
Some upstream users requested specifically `failed`